MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -116,24 +116,10 @@ open class BaseInstructionsBannerView: UIControl {
     }
     
     /**
-     Updates the instructions banner for a given `RouteProgress`.
+     Updates the instructions banner distance info for a given `RouteStepProgress`.
      */
-    public func update(for currentLegProgress: RouteLegProgress) {
-        let stepProgress = currentLegProgress.currentStepProgress
-        let distanceRemaining = stepProgress.distanceRemaining
-        
-        guard let visualInstructions = stepProgress.remainingVisualInstructions else { return }
-        
-        for visualInstruction in visualInstructions {
-            if stepProgress.distanceRemaining <= visualInstruction.distanceAlongStep || stepProgress.visualInstructionIndex == 0 {
-                
-                set(visualInstruction)
-                
-                stepProgress.visualInstructionIndex += 1
-                break
-            }
-        }
-        
+    public func updateDistance(for currentStepProgress: RouteStepProgress) {
```

Comment:
I think thus function should be removed and combined with `Set(_:)`.

Reply:
@bsudekum we do not want to combine this with `Set(_:)`. It is crucial that we invoke this everytime we receive a location update.